sklearn/cluster/hierarchical.py

```diff
@@ -783,9 +796,11 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         By default, no caching is done. If a string is given, it is the
         path to the caching directory.
 
-    n_components : int, optional
```

Comment:
I think we should have actually removed all these docstrings, as we usually do when deprecating... I was to fast to merge...